scripts/us_census/pep_srh/process.py

```diff
@@ -0,0 +1,1057 @@
+    df.to_csv(output_file_path + output_file_name, index=False)
+
+
+def process_national_files_1990_2000():
+    '''
+    Process for National Level data, 1990 - 2000
+    Uses State Level file as input
+    '''
+    # Section 1 - Writing As Is Data
+    input_file_path = _CODEDIR + PROCESS_AS_IS_DIR + '1990_2000/state/'
+    output_file_path = _CODEDIR + PROCESS_AS_IS_DIR + '1990_2000/national/'
+    output_file_name = 'national_1990_2000.csv'
+    files_list = os.listdir(input_file_path)
+    files_list.sort()
+
+    df = pd.read_csv(input_file_path + files_list[0])
+    df['LOCATION'] = 'country/USA'
+    df = df.groupby(['YEAR', 'LOCATION']).agg('sum').reset_index()
+    df.to_csv(output_file_path + output_file_name, index=False)
+
+    # Section 2 - Writing Agg data
+    input_file_path = _CODEDIR + PROCESS_AGG_DIR + '1990_2000/state/'
+    output_file_path = _CODEDIR + PROCESS_AGG_DIR + '1990_2000/national/'
+    files_list = os.listdir(input_file_path)
+    files_list.sort()
+
+    df = pd.read_csv(input_file_path + files_list[0])
+    df['LOCATION'] = 'country/USA'
+    df = df.groupby(['YEAR', 'LOCATION']).agg('sum').reset_index()
+    df.to_csv(output_file_path + output_file_name, index=False)
+
+
+def process_national_files_2000_2010():
+    '''
+    Process for National Level data, 2000 - 2010
+    Uses State Level file as input
+    '''
+    # Section 1 - Writing As Is Data
+    input_file_path = _CODEDIR + PROCESS_AS_IS_DIR + '2000_2010/state/'
+    output_file_path = _CODEDIR + PROCESS_AS_IS_DIR + '2000_2010/national/'
+    output_file_name = 'national_2000_2010.csv'
+    files_list = os.listdir(input_file_path)
+    files_list.sort()
+
+    df = pd.read_csv(input_file_path + files_list[0])
+    df['LOCATION'] = 'country/USA'
+    df = df.groupby(['YEAR', 'LOCATION']).agg('sum').reset_index()
+    df.to_csv(output_file_path + output_file_name, index=False)
+
+    # Section 2 - Writing Agg data
+    output_file_path = _CODEDIR + PROCESS_AGG_DIR + '2000_2010/national/'
+    df1 = pd.DataFrame()
+    df1['YEAR'] = df['YEAR'].copy()
+    df1['LOCATION'] = df['LOCATION'].copy()
+    df1['NH'] = (df['NH_MALE'] + df['NH_FEMALE']).copy()
+    df1['NHWA'] = (df['NHWA_MALE'] + df['NHWA_FEMALE']).copy()
+    df1['NHBA'] = (df['NHBA_MALE'] + df['NHBA_FEMALE']).copy()
+    df1['NHIA'] = (df['NHIA_MALE'] + df['NHIA_FEMALE']).copy()
+    df1['NHAA'] = (df['NHAA_MALE'] + df['NHAA_FEMALE']).copy()
+    df1['NHNA'] = (df['NHNA_MALE'] + df['NHNA_FEMALE']).copy()
+    df1['NHTOM'] = (df['NHTOM_MALE'] + df['NHTOM_FEMALE']).copy()
+    df1['H'] = (df['H_MALE'] + df['H_FEMALE']).copy()
+    df1['HWA'] = (df['HWA_MALE'] + df['HWA_FEMALE']).copy()
+    df1['HBA'] = (df['HBA_MALE'] + df['HBA_FEMALE']).copy()
+    df1['HIA'] = (df['HIA_MALE'] + df['HIA_FEMALE']).copy()
+    df1['HAA'] = (df['HAA_MALE'] + df['HAA_FEMALE']).copy()
+    df1['HNA'] = (df['HNA_MALE'] + df['HNA_FEMALE']).copy()
+    df1['HTOM'] = (df['HTOM_MALE'] + df['HTOM_FEMALE']).copy()
```

Comment:
For such repeated blocks with the same pattern, can use list:
```
for p in ['NH', 'NHWA', 'NHBA', 'NHIA', ....]:
     df1[p] = (df[p+'_MALE'] + df[p+'_FEMALE']).copy()
```

Reply:
changed to include agg columns from list